src/serializers/StackExchange.Redis.Extensions.MemoryPack/StackExchange.Redis.Extensions.MemoryPack.csproj

```diff
@@ -1,25 +1,17 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net7.0</TargetFrameworks>
```

Comment:
Would it be better to have this set via Directory.Build.props

Reply:
I don't know why it's claimed here. I've removed it and current tests are passed. But still require review from @imperugo .

Reply:
By removing the property in the csproj it inherits from directory.build.props.